apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,28 @@
-<div
-  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
-  [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+     [class]="customClass">
+
+  <ng-content select="[cardImage]"></ng-content>
 
   <section>
-    <app-list-item
-      *ngFor="let item of list"
-      [name]="item.firstname"
-      [id]="item.id"
-      [type]="type">
+    <app-list-item *ngFor="let item of list">
+
+      <ng-container name>
+        {{item.firstname ?? item.name}}
+      </ng-container>
+
+      <ng-container deleteBtn *ngTemplateOutlet="deleteBtnTemplate; context:{$implicit: item.id}"></ng-container>
+
```

Comment:
use ngTemplateOutlet combine with ContentCHild to get a custom item from your parent to make it more customizable.

what happen if now you have to add an icon but only for citycard?